features/notifications/useNotifications.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from "react-query";
+import type { User } from "../users/UserModels";
+import axios from "axios";
+import type { TNotification } from "./NotificationsModels";
+
+export function useNotifications(userId: User["id"], seen: string, dependencies: any[] = []) {
```

Comment:
We already have useNotifications as toaster service.

Reply:
Yes, and that can bother us when importing existing useNotifications. I don't like this solution but I will rename it to useTNotifications as it is used only in the notifications folder scope.